Wrong path to HMR runtime

When enabling HMR in the server's host app, I get the following error:

```
ERROR in webpack/runtime/readFile chunk loading
Cannot find module '../hmr/JavascriptHotModuleReplacement.runtime.js'
```

Looking at the code, I saw the path used to require the HMR runtime is relative, while the other requires for webpack rely directly on the dependency. I debugged the code at [this point](https://github.com/module-federation/node/blob/7efd6f162851850de13002778035418544a91cf8/src/LoadFileChunkLoadingRuntimeModule.js#L354-L356) and the current working directory is the root of my project so the relative path won't work.

I made the change to rely on the dependency instead of the relative path and it worked, so I have created [this PR](https://github.com/module-federation/node/pull/8) with the changes.

#### Additional info

Some details about my use case.

I have an app that runs webpack in the client and server, both with HMR enabled.
The app installs a package with common scripts intended to be used across apps, like running webpack in dev mode.
I have a remote module configured for this app.

I get the error when running webpack in dev mode with HMR in the server enabled.

--- a/src/LoadFileChunkLoadingRuntimeModule.js
+++ b/src/LoadFileChunkLoadingRuntimeModule.js
@@ -352,7 +352,7 @@ class ReadFileChunkLoadingRuntimeModule extends RuntimeModule {
             '}',
             '',
             Template.getFunctionContent(
-              require('../hmr/JavascriptHotModuleReplacement.runtime.js')
+              require('webpack/lib/hmr/JavascriptHotModuleReplacement.runtime.js')
             )
               .replace(/\$key\$/g, 'readFileVm')
               .replace(/\$installedChunks\$/g, 'installedChunks')

